src/common/snippets/CMakeLists.txt

```diff
@@ -45,3 +45,7 @@ set_target_properties(${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_REL
 # install
 
 ov_install_static_lib(${TARGET_NAME} ${OV_CPACK_COMP_CORE})
+
+if(NOT ENABLE_TESTS)
+    openvino_developer_export_targets(COMPONENT ${OV_CPACK_COMP_CORE} TARGETS ${TARGET_NAME})
```

Comment:
why do we need condition for ENABLE_TESTS?

Reply:
The condition was needed so as to avoid exporting `inference_engine_snippets` twice.
Exporting the library both with `core` and `tests` components leads to CMake configuration errors:
```
CMake Error at /agent/_work/1/openvino/cmake/developer_package/message.cmake:20 (_message):
  Some (but not all) targets in this export set were already defined.

  Targets Defined: openvino::inference_engine_snippets
```

Removed both the condition and export to `tests` - always exporting the library only with the `core` component seems like a better idea. 
Could you please take a look one more time?

Reply:
I suppose exporting this library as a core component is a proper solution, since it's a core library, not a test component.